arrowjs/docs/table-of-contents.json

```diff
@@ -1,57 +1,61 @@
 {
-  "id": "table-of-contents",
+  "id": "table-of-contents-arrow",
```

Comment:
So the TOCs just get concatenated in the order they are mentioned?

Reply:
Yes.